### PR TITLE
Update feedstock for 6.6.0

### PR DESCRIPTION
**Pull request by honnibal**
Fingers crossed on this -- this might be complicated because I'm now compiling some CUDA kernels as well.

### CODE REVIEW

**Comment by rolando** [comment]
@honnibal This customization in setup.py: https://github.com/explosion/thinc/blob/master/setup.py#L55
fails on windows. Is CUDA support for `thinc` supported on Windows?

We have these alternatives:

* Add `skip: True  # [win]` under build section so we don't build for windows for now.
* Apply a patch via `build.patches` section so we can build a windows package for this release skipping the CUDA part. https://github.com/explosion/thinc/blob/master/setup.py#L108
* Wait this issue be fixed in upstream and publish next release.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,8 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256sum }}
-  patches:
-    - patches/0001-Fix-integer-check.patch
 
 build:
   number: 0
```
Comment (rolando) [inline recipe/meta.yaml]: You can add a new line

`    skip: True  # [win]`

To skip the windows builds for this release until the setup issue is fixed.

Comment (honnibal) [inline recipe/meta.yaml]: I'd rather fix in upstream and make a 6.6.1. I'll think about how to do this.

Thinc doesn't require CUDA -- it should degrade to CPU if it's not available. It's just a question of getting the setup.py logic right.

Comment (rolando) [inline recipe/meta.yaml]: Sounds good to me 👍

**Pull request closed without merging**